frontend/common/providers/IdentityProvider.js

```diff
@@ -121,4 +121,4 @@ IdentityProvider.propTypes = {
   onSave: OptionalFunc,
 }
 
-module.exports = IdentityProvider
+export default IdentityProvider
```

Comment:
I think we should use this approach for all the default exports right @kyle-ssg ?

Reply:
Yes, doing it gradually as we introduce changes/new functionality

Reply:
The PR is still WIP btw as there's a lot of work - getting it to a point where e2e is passing first